README.mdown

```diff
@@ -1,131 +1,211 @@
-# Generate Dependent Styles
-The script `generate_styles.rb` can be used to automatically generate dependent CSL styles for publishers that use the same citation format(s) across multiple journals. All that is needed is a collection of journal metadata (journal title, ISSN, eISSN, citation format, etc.).
+#### Notice
+This repository was created on 2016-03-10 as a dedicated home for the journal metadata previously stored at https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles.
```

Comment:
I guess that the url will not exist for long anymore. Therefore I suggest to write:
"for the journal metadata previously part of https://github.com/citation-style-language/utilities/ ."

(Alternatively, you could also link to a specific version, e.g. https://github.com/citation-style-language/utilities/tree/65d1c5ccb6c9fc5faa539f26e24e9f1e13e67b8e/generate_dependent_styles but I think this makes it more confusing...)